patches/minecraft/net/minecraft/world/level/block/StemBlock.java.patch

```diff
@@ -11,16 +11,13 @@
 +         if (net.minecraftforge.common.ForgeHooks.onCropsGrowPre(p_222539_, p_222540_, p_222538_, p_222541_.m_188503_((int)(25.0F / f) + 1) == 0)) {
              int i = p_222538_.m_61143_(f_57013_);
              if (i < 7) {
--               p_222538_ = p_222538_.m_61124_(f_57013_, Integer.valueOf(i + 1));
--               p_222539_.m_7731_(p_222540_, p_222538_, 2);
-+               p_222539_.m_7731_(p_222540_, p_222538_.m_61124_(f_57013_, Integer.valueOf(i + 1)), 2);
-             } else {
+                p_222538_ = p_222538_.m_61124_(f_57013_, Integer.valueOf(i + 1));
+@@ -55,11 +_,12 @@
```

Comment:
Why is this being reverted? This looks to affect the `ForgeHooks.onCropsGrowPost` call below.

Reply:
My appologies, I thought I didn't commit this change, since like you said, it changes the event call. I probably forgot to run patches.